src/client/src/ui/spotTile/components/TileBusinessLogic.ts

```diff
@@ -14,19 +14,29 @@ const MIN_RFQ_VALUE = 10000000
 const RESET_NOTIONAL_VALUE = DEFAULT_NOTIONAL_VALUE
 
 // Utils
-export const getDefaultNotionalValue = () => numeral(DEFAULT_NOTIONAL_VALUE).format(NUMERAL_FORMAT)
+export const getDefaultNotionalValue = (currencyPair: CurrencyPair) =>
+  // This is to simply to have one Tile showing RFQ prompt on page load
+  // check JIRA ticket ARTP-532
+  currencyPair.symbol === 'NZDUSD'
+    ? numeral(MIN_RFQ_VALUE).format(NUMERAL_FORMAT)
+    : numeral(DEFAULT_NOTIONAL_VALUE).format(NUMERAL_FORMAT)
+
 export const getNumericNotional = (notional: string) =>
   numeral(notional).value() || DEFAULT_NOTIONAL_VALUE
+
 export const isValueInRfqRange = (notional: string) => {
   const numericValue = convertNotionalShorthandToNumericValue(notional)
   return numericValue >= MIN_RFQ_VALUE && numericValue <= MAX_NOTIONAL_VALUE
 }
+
 const isValueOverRftRange = (notional: string) => {
```

Comment:
Just a minor typo:  should be ```isValueOverRfqRange``` rather than ```isValueOverRftRange```

Reply:
Oups! Fixed ;-)